pipert/contrib/vid_capture.py

```diff
@@ -19,6 +19,8 @@ def __init__(self, endpoint, stream_address, out_key, redis_url, fps=30.0, maxle
         t_upload = Frames2Redis(out_key, redis_url, self.queue, maxlen, name="upload_redis").as_thread()
         self.register_routine(t_upload)
 
+    def changeStream(self, stream_address, fps=30.0):
+        self._routines[0].updatedConfig = {"stream_address": stream_address, "FPS": fps}
```

Comment:
Agreed. 
Maybe we can save t_stream and t_upload as instance attributes of VidCapture? (though we'd still have to register them to the routine list)
Or maybe instead of a list we could use a dictionary where the keys are the routine names?